test/e2e-tests.sh

```diff
@@ -109,6 +109,13 @@ function dump_extra_cluster_state() {
   kubectl logs $(get_app_pod controller knative-serving)
 }
 
+function publish_test_images() {
+  image_dirs="$(find ${REPO_ROOT_DIR}/test/test_images -mindepth 1 -maxdepth 1 -type d)"
+  for image_dir in ${image_dirs}; do
+    ko publish -P "github.com/knative/serving/test/test_images/$(basename ${image_dir})"
```

Comment:
It is notable that this will not work with DockerHub (and possible other registries), which is why I'd changed the `ko` default behavior :(

Reply:
Shall I fix this separately?